### ISSUE TITLE
[Feature Request] Check for valid semver

### ISSUE THREAD

**Issue by stefanklug** [jubianchi/semver-check#94]
Hi,

I just stumbled open the project. Thanks a lot for your work. I was looking for a way to validate if a given string is a valid semantic version at all. It would be great if the checker could reject illegal semantic versions. Examples would be:

- 1.0 (patch missing)
- 1.0.0+a+a (Meta contains a + sign)

**Comment by slowr** [comment]
Kinda does it already, if it cannot tell you the next version then the version is illegal 🙄 

Maybe they can add an alert that is invalid

**Comment by jubianchi** [comment]
Hi! I will have a look at your request soon.

**Comment by foobarbecue** [comment]
Yeah... this is the first google result for semver checker, so it really should check semver correctly, but it doesn't. It was wrong on many cases I checked.

https://semver.org provides [a regular expression](https://regex101.com/r/Ly7O1x/3/) that you should use.
